apps/events/models.py

```diff
@@ -311,7 +312,8 @@ def _trigger(self):
         utc_now = timezone.now()
         self.last_triggered_at = utc_now
         self.total_triggers += 1
-        if self.total_triggers >= self.params["repetitions"]:
+        repetitions = self.params["repetitions"]
+        if (repetitions and self.total_triggers >= repetitions) or (self.end_date and self.end_date >= timezone.now()):
```

Comment:
I think I might have accidentally supported infinite reminders. If the user ask the bot to remind them daily about something for instance, the bot will set the `end_date` and repetitions to `null` (according to my test results). Since this branch will be entered when only when `repetitions` or `end_date` is not None, this it will thus never be entered, resulting in infinite reminders